test/duct/middleware/web_test.clj

```diff
@@ -19,7 +19,16 @@
         (is (= (handler (mock/request :get "/")) response))
         (is (= @logs [[:info :duct.middleware.web/request
                        {:request-method :get, :uri "/"}]]))))
-    
+
+    (testing "synchronous with log level"
+      (let [logs     (atom [])
+            handler  (wrap-log-requests (constantly response)
+                                        (->TestLogger logs)
+                                        :trace)]
```

Comment:
As per my last comment, could you fix the spacing in the `let` clause? There are 2 spaces after `handler` instead of 1. Otherwise it looks fine.